config/routes.rb

```diff
@@ -1,2 +1,7 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1, defaults: { format: :json } do
```

Comment:
I really don't see any reason for versioning this. But it's not a blocker.

Reply:
ok